soleBasket/src/components/NavBar/NavBar.jsx

```diff
@@ -1,8 +1,30 @@
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import "./NavBar.css"
 import "./SearchBar.css"
+import {useData} from "../../context/dataContext"
```

Comment:
Try using index file for context, it reduces the number of imports

Reply:
sure! will keep that in mind!